patches/youch@4.1.0-beta.10.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/build/chunk-4XB2BYKC.js b/build/chunk-4XB2BYKC.js
+index 6e6e490115bc4b107b53834ceac3d7d19cdbe3fe..37373d734d034caf45e28c1b165e5c176f901137 100644
+--- a/build/chunk-4XB2BYKC.js
++++ b/build/chunk-4XB2BYKC.js
+@@ -34,10 +34,10 @@ var ErrorStackSource = class extends BaseComponent {
+    */
+   async toHTML(props) {
+     const frame = props.frame;
+-    if (frame.type === "native" || !frame.source || !frame.fileName) {
++    if (!frame.source || !frame.fileName) {
+       return "";
+     }
+-    const language = LANGS_MAP[extname(frame.fileName)] ?? "plain";
++    const language = LANGS_MAP[extname(frame.fileName)] ?? (frame.type === "native" ? "js" : "plain");
```

Comment:
Native modules have no extension so there is no syntax highlight by default. But it should be safe to assume native modules to be javascript if available?